lib/ui/common/RegExInputFormatter.dart

```diff
@@ -0,0 +1,45 @@
+import 'package:flutter/services.dart';
+
+class RegExInputFormatter implements TextInputFormatter {
```

Comment:
I think that this class should go to a utils/ folder.

Reply:
Done